### ISSUE TITLE
Works on Simulator but not real device.

### ISSUE THREAD

**Issue by rclayborn** [zappycode/Pokemon#1]
Hi Nick, I am trying to figure out why this app is NOT centering on real device. but I am having a bit of trouble. I took your course and finished it. Now I am trying to app it to my own app. 
I made a bicycle touring app (caught the flag type of app) using this model but I can't get it to work on my "REAL" device. so I downloading your git hub version and it has the same problem. I thought you may want to know. CHEERS!

**Comment by zappycode** [comment]
Hey there! Looks like there were a few changes that needed to be made for it to work with the public version of Xcode 8 so I went and committed those. As far as the centering stuff goes. I just tried it on the simulator and on my device and it worked. Are you still having issues? If so let me know and we can open this back up.

**Comment by rclayborn** [comment]
Thank all it well and thanks so much. I was going to go on today and take my commit off. Not sure what MY issue was.Randall 

```
On Monday, October 3, 2016 10:42 AM, Zappy Code <notifications@github.com> wrote:
```

 Hey there! Looks like there were a few changes that needed to be made for it to work with the public version of Xcode 8 so I went and committed those. As far as the centering stuff goes. I just tried it on the simulator and on my device and it worked. Are you still having issues? If so let me know and we can open this back up.—
